.travis.yml

```diff
@@ -62,6 +63,14 @@ script:
          git checkout -qf $TRAVIS_COMMIT
        fi &&
        . /root/.opam/opam-init/init.sh &&
+       git submodule update --init sub/coq &&
+       if [ $COQ_BRANCH != "" ]
+       then ( cd sub/coq && 
+       	      git checkout $COQ_BRANCH )
+       fi &&
+       ( cd sub/coq && 
+         git branch &&
+	 git describe --dirty --long --always --abbrev=40 --tags ) &&
```

Comment:
The indentation is off at this line.

Reply:
Well, tabs and spaces have different semantics. This is equivalent to claiming randomly replacing $0$ by $\emptyset$ is fine because they will probably be the same set in ZFC.

Reply:
TAB and SPACE have equivalent semantics in the vast majority of languages.  The only ones I can think of where that's not true are TeX and make.  Anyway, I've removed the tabs, so we'll see if that's a problem here.